### DISCUSSION TITLE
How do I use local storage?

### DISCUSSION THREAD

**Discussion by vbezhenar** [cloudnative-pg/cloudnative-pg#1443] [Q&A]
I have a kubernetes 1.24 cluster with two nodes. I want to run postgres cluster with two pods. Those pods should be scheduled on each node. And I want to use node local disk as storage. I'm having trouble to understand how should I configure cloud native pg to accomplish that. Is it possible? I tried to use network attached volumes with ordinary pvc and it works fine, but local disk performance is much better.

**Comment by ns-jhu** [comment]
follow readme file in git@github.com:kubernetes-sigs/sig-storage-local-static-provisioner.git
